tests/registration_test.py

```diff
@@ -533,6 +533,52 @@ def test_strings_groupby_attach(self):
         self.assertIsInstance(sAttach.segments, ak.pdarray)
         self.assertIsInstance(sAttach.unique_keys, ak.Strings)
 
+    def test_string_property_accessor(self):
+        cleanup()
+        s = ak.array(["one", "two", "three", "123"])
+        bytes_answers = ak.array(
+            [111, 110, 101, 0, 116, 119, 111, 0, 116, 104, 114, 101, 101, 0, 49, 50, 51, 0]
+        )
+        offsets_answers = ak.array([0, 4, 8, 14])
+
+        # Testing for registration after being accessed and being removed after unregistering s
+        # Also tests for correct values in accessed arrays
+        s.register("sname")
+        self.assertTrue(len(ak.list_registry()) == 1)
+
+        sbytes = s.mybytes
+        self.assertListEqual(sbytes.to_list(), bytes_answers.to_list())
+        self.assertTrue(len(ak.list_registry()) == 2)
+
+        soffsets = s.myoffsets
+        self.assertListEqual(soffsets.to_list(), offsets_answers.to_list())
+        self.assertTrue(len(ak.list_registry()) == 3)
+
+        s.unregister()
+        self.assertTrue(len(ak.list_registry()) == 0)
+
+
+        # Tests for unregistering each property
```

Comment:
you don't need two new lines here, if u run `black` on this file and it should reformat it correctly

Reply:
Updated